modules/scalaz/src/main/tut/README.md

```diff
@@ -29,27 +29,33 @@ import com.typesafe.config.ConfigFactory.parseString
 import pureconfig._
 import pureconfig.generic.auto._
 import pureconfig.module.scalaz._
-import scalaz.{==>>, IList, ISet, Maybe, NonEmptyList}
+import scalaz.{ ==>>, DList, Dequeue, IList, ISet, Heap, Maybe, NonEmptyList }
 import scalaz.std.anyVal.intInstance
 import scalaz.std.string._
 
 case class ScalazConfig(
-  numberLst: IList[Int],
+  numberDlist: DList[Int],
+  numberDequeue: Dequeue[Int],
+  numberIlist: IList[Int],
+  numberHeap: Heap[Int],
   numberSet: ISet[Int],
+  numberMaybe: Maybe[Int],
   numberNel: NonEmptyList[Int],
-  numberMap: String ==>> Int,
-  numberMaybe: Maybe[Int]
+  numberMap: String ==>> Int
 )
 ```
 
 We can read a `ScalazConfig` like:
 ```tut:book
 val scalazConf = parseString("""{
-  number-lst: [1,2,3],
+  number-dlist: [1,2,3],
```

Comment:
Regarding your error, I'd actually leave a single list-like structure here and delete the others. There is little value in showing all the data structures we support if the mechanism to encode and read them is always the same.